test/Macros/SwiftifyImport/CxxSpan/LifetimeboundSpan.swift

```diff
@@ -48,3 +53,8 @@ func myFunc4(_ vec: borrowing VecOfInt, _ span: SpanOfInt) -> SpanOfInt {
 // CHECK-NEXT: func myFunc4(_ vec: borrowing VecOfInt, _ span: Span<CInt>) -> Span<CInt> {
 // CHECK-NEXT:     return Span(_unsafeCxxSpan: myFunc4(vec, SpanOfInt(span)))
 // CHECK-NEXT: }
+
+// CHECK:      @_alwaysEmitIntoClient @lifetime(borrow self)
```

Comment:
How does `@lifetime(borrow self)` work when attached to a free function?

Reply:
In a free function it would trigger a diagnostic error. In this case it is not really a free function, but the dump does not include the enclosing type.

Reply:
Oh right, I totally missed the struct